### PR TITLE
Setting up Flake8

### PR DESCRIPTION
**Pull request by MatanP12**
- Installing Flake8 to the Vagrant VM.
- Adding Flake8 as GitHub Action.

Issue: #5

### CODE REVIEW

**Review by OmerCS8 — APPROVED** [review]
LGTM

**Review by TomerNewmanPrograms — APPROVED** [review]
Looks Ok

**Comment by MatanP12** [comment]
> You have linked the PR to itself, it should be fix #5

My bad, fixed.

File: .github/workflows/flake8.yml [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+    container: fedora:34
+    steps:
+        - name: Install Pipenv and Git
+          run: dnf install -y pipenv git
+        - name: Checkout code
+          uses: actions/checkout@v2
+        - name: Setup environment
+          run: pipenv sync --dev
+        - name: Run Flake8
+          run: pipenv run flake8 --max-line-length 120
```
Comment (natifridman) [inline .github/workflows/flake8.yml:16]: YAML is an indentation sensitive language. Please use 2 spaces here and not a tab or 4 spaces

**Review by natifridman — CHANGES_REQUESTED** [review]

**Review by natifridman — APPROVED** [review]
Approved. Please squash your commits

**Review by natifridman — APPROVED** [review]

**Pull request merged**